test-files/test_transformer.py

```diff
@@ -0,0 +1,53 @@
+"""Testing files for transformer.py
+"""
+
+#Import transformer.py and imbedded modules
+import argparse
+import transformer
+import transformer_class
+
+#Setting up some testing classes
+TEST_TRANSFORMER = transformer_class.Transformer()
+PARSE = argparse.ArgumentParser()
+
+# pylint: disable=assignment-from-no-return
+def test_add_parameters():
+    """Test for add_parameters function
+    """
+
+    #Saving method call to variable
```

Comment:
There should be a space between the `#` and the first following character. Here and elsewhere